tests/integration/devfile/cmd_devfile_registry_test.go

```diff
@@ -81,6 +83,30 @@ var _ = Describe("odo devfile registry command tests", func() {
 			helper.CmdShouldPass("odo", "registry", "delete", registryName, "-f")
 			helper.CmdShouldFail("odo", "create", "java-maven", "--registry", registryName)
 		})
+	})
 
+	Context("when working with git based registries", func() {
+		It("should show deprecation warning when the git based registry is used", func() {
+			deprecated := "Deprecated"
```

Comment:
IMO we should have common steps in `JustBeforeEach`, no?

Reply:
I am not sure if it makes a difference at this point as there is only one `It` block

Reply:
> I am not sure if it makes a difference at this point as there is only one `It` block

Only one you mean? I guess there are two but correct me if I am wrong.